nixos/modules/services/networking/shout.nix

```diff
@@ -33,14 +48,35 @@ in {
 
     configFile = mkOption {
       type = types.nullOr types.lines;
-      default = null;
+      default = configFile;
       description = ''
         Contents of Shout's <filename>config.js</filename> file. If left empty,
-        Shout will generate from its defaults at first startup.
+        It will generated from the <literal>config</literal> attributes, merged
```

Comment:
@joachifm It totally makes sense, I forgot to remove the `nullOr` type. I updated the PR, thanks a lot for your input.

I'm not sure how to word the description though, I'm introducing the `config` option in this PR to make it easier to configure, and I keep `configFile` mostly for backward compatibility. I updated it to mention this and recommend using `config`.